src/com/company/Players.java

```diff
@@ -0,0 +1,21 @@
+package com.company;
+
+public class Players implements IPlayers {
```

Comment:
Why is this called Players when it denotes a single player?

Reply:
ok, It should be Player